models/public/fast-neural-style-mosaic-onnx/fast-neural-style-mosaic-onnx.md

```diff
@@ -0,0 +1,106 @@
+# fast-neural-style-mosaic-onnx
```

Comment:
Please add this model to the index (`models/public/index.md`).

Reply:
Added, it is located after Image Inpainting